src/lib/leagues.ts

```diff
@@ -0,0 +1,135 @@
+import { League } from "./api/models/League"
+
+type TierInformation = {
+  minPoint: number
+  nextLeague?: League
+  nextTier?: Tier
+}
+
+export enum Tier {
+  ONE = 1,
+  TWO = 2,
+  THREE = 3,
+}
+
+type LeagueDetails = Record<Tier, TierInformation>
+
+export const leagueInformations: Record<League, LeagueDetails> = {
+  aspirant: {
+    3: {
+      minPoint: 0,
+      nextLeague: League.ASPIRANT,
+      nextTier: 2,
+    },
+    2: {
+      minPoint: 1028,
```

Comment:
1. how did you get these ranges?
2. do we know if they are static or if they constantly shift?
3. how are we going ensure to keep them up to date? (if there's an automated way to get these, I'd move the logic to API to make sure they are always up to date)

Reply:
They were collected by players on Discord https://discord.com/channels/1101590942076653660/1203506066919391232/1203506066919391232.

It seems that they are static for now since it didn't change from the beginning of the beta phase. But it is true that there were no official communication about it so we cannot be 100% sure.

Keeping the values up to date will have to be done manually for now as I don't see any automated way to get those (it might change in the next beta build). But maybe it will be provided by FG through an API endpoint in the future? 

It is not ideal but I was thinking that it was good enough for a first iteration since it is pretty interesting data to see when you want to farm the ladder (in-game it tells you the number of points you need to go to the next tier/league but you cannot visualize your progress in that league/tier).